.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
```

Comment:
What about the `schedule`? Is it valid to put an `if:` under the `- cron:` like you can do for `- step:`? VSCode is telling me this isn't allowed, but it could just not be in the know.

It's also telling me the `${{ }}` is not allowed for the `fail-fast` strategy

Reply:
Is there a reason to not always `fail-fast: true`?

Reply:
I'm going to try this on another branch to always run this on main _or_ when the event_name isn't `schedule`:

```yml
jobs:
  build:
    if: ${{ github.ref == 'refs/heads/main' || github.event_name != 'schedule' }}
```

Reply:
Nevermind, it seems `schedule` only operates on the main project branch:

> Scheduled workflows run on the latest commit on the default or base branch. 

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
Separate PR for this: https://github.com/articulate/docker-articulate-node/pull/87